packages/tools/devtools/devtools-core/src/Features.ts

```diff
@@ -51,4 +51,20 @@ export interface ContainerDevtoolsFeatureFlags {
 	 * Indicates that the Container Devtools supports visualizing the data associated with the Container.
 	 */
 	containerDataVisualization?: boolean;
+
+	/**
+	 * Indicates that the Container Devtools supports modifying the container's connection and lifecycle state.
+	 *
+	 * @remarks
+	 * When `true`, the devtools instance supports:
+	 * - **Connect**: Connecting a disconnected container
+	 * - **Disconnect**: Disconnecting a connected container
+	 * - **Close**: Permanently closing/disposing the container
+	 *
+	 * When `false`, these operations are not available (typical for data object devtools
+	 * which don't have direct control over container lifecycle).
+	 *
+	 * If this property is `undefined`, it should be treated as `true` by consumers.
```

Comment:
If there is no setup for this feature flag, we default to assume consumer has the ability to modify the connection state here sounds questionable to me. But this is not affecting the code change here.